### PR TITLE
Add initial version of the logo

### PR DESCRIPTION
**Pull request by Wes1262**

### CODE REVIEW

File: logo_purple.svg [hunk 1]
```diff
@@ -0,0 +1,1349 @@
...
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   style="fill-rule:evenodd"
+   id="svg7046"
+   version="1.1"
+   viewBox="0 0 1350 1350"
+   height="1800"
+   width="1800">
```
Comment (kelunik) [inline logo_purple.svg:13]: Why is the width and height different from the viewbox size?

Comment (Wes1262) [reply logo_purple.svg:13]: I have no idea. Inkscape did it probably. I don't know much about SVG.

The natural width is 1800 so I can change that. Wait for it...

Comment (kelunik) [reply logo_purple.svg:13]: Don't know either, just had a look at the raw contents. :smile:

**Comment by kelunik** [comment]
@Netmosfera You wanted to add a PR with just the current SVG and a PNG, right?

**Pull request merged**